### PR TITLE
vimc-7192 - alter table coverage - add percent_gavi_support

### PR DESCRIPTION
**Pull request by xiangli313**
Youtrack: 
https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-7192/db-migration-create-percentgavisupport

Task:
Gavi requested adding % of gavi_support in future data sets. Hence, we record this % in coverage table.

### CODE REVIEW

File: migrations/sql/V2023.08.21.1254__AddPercentGaviSupportToCoverage.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- To carry out new impact calculations we need to indicate
+-- percentage of GAVI support in a given vaccination activity.
+ALTER TABLE coverage
+    ADD COLUMN percent_gavi_support DECIMAL DEFAULT NULL
```
Comment (richfitz) [inline migrations/sql/V2023.08.21.1254__AddPercentGaviSupportToCoverage.sql:4]:
```suggestion
    ADD COLUMN percent_gavi_support DOUBLE PRECISION DEFAULT NULL
```

The "decimal" type has much worse performance. I see we still have a bunch of those around though. It's not super obvious which is which typically!

https://www.postgresql.org/docs/current/datatype-numeric.html

**Review by richfitz — CHANGES_REQUESTED** [review]
Unless there's a good reason not to, please use the simpler type.

Note that there will be a delay on getting this deployed as we are in the process of moving servers around

**Review by richfitz — CHANGES_REQUESTED** [review]
Unless there's a good reason not to, please use the simpler type.

Note that there will be a delay on getting this deployed as we are in the process of moving servers around

**Comment by xiangli313** [comment]
Thanks, Rick. I will give you a reminder after the process of moving servers.

File: migrations/sql/V2023.08.21.1254__AddPercentGaviSupportToCoverage.sql
Comment (richfitz) [inline migrations/sql/V2023.08.21.1254__AddPercentGaviSupportToCoverage.sql:1]: when we merge this, update the filename to be later than this, as I've set other migrations to run first

**Review by richfitz — CHANGES_REQUESTED** [review]

**Review by richfitz — APPROVED** [review]

**Pull request merged**